pandas/core/indexes/datetimes.py

```diff
@@ -1707,7 +1707,64 @@ def freq(self, value):
     nanosecond = _field_accessor('nanosecond', 'ns',
                                  "The nanoseconds of the datetime")
     weekofyear = _field_accessor('weekofyear', 'woy',
-                                 "The week ordinal of the year")
+    """
```

Comment:
This is where `E128` is still complaining a bit. I don't know what the nicest way is to resolve this since we're a bit unconventional with the `_field_accessor` accepting a docstring here. I could add tabs to the docstring, but it would make the code much less readable (as well cause ugly things to the docstring).

Reply:
@TomAugspurger would love advice.

Reply:
@koaning maybe just make a new variable called `_weekofyear_doc` and assign it to like

        weekofyear = _field_accessor('weekofyear', 'woy', _weekofyear_doc)

Reply:
you'll get a lot of variables in that file, but that works for me. will make the change. thanks!